src/Sylius/Bundle/ShopBundle/Resources/views/Account/dashboard.html.twig

```diff
@@ -18,11 +18,25 @@
             {{ customer.fullName }}
         </div>
         <div class="ui item">
-            <strong>{{ customer.email }}</strong>
+            <strong>
+                {{ customer.email }}
```

Comment:
Just for further reference, we make create `User::isVerified()` method to use `{% if customer.user.verified %}` instead etc.